### DISCUSSION TITLE
Dynamic Resolution not working (Guest: Linux Mint Xfce) Intel Mac

### DISCUSSION THREAD

**Discussion by mkronenberg** [utmapp/UTM#4678] [UTM Help (macOS)]
Dynamic Resolution not working, My guest VM is Linux Mint Xfce, my host machine is an Intel Mac

I installed the SPICE agent
https://docs.getutm.app/guest-support/linux/#spice-agent

I also followed the troubleshooting step
https://docs.getutm.app/guest-support/dynamic-resolution/
Weird behavior when I run the `$ xrandr --output Virtual-1 --auto
` command. The window will enlarge by roughly 10%. That behavior repeats for each additional entry of the command. Window size resets on the next opening of the VM

**Comment by meop** [comment]
I found the same behavior for XFCE. Clipboard worked but not Dynamic Resize, even with the xrandr command, it does not consistently help.

KDE (Plasma) worked ok, except for this issue: https://github.com/utmapp/UTM/issues/4064

Edit: Sorry, this is the issue affecting SPICE and non-GNOME desktops: https://github.com/systemd/systemd/issues/18791

Although, the UTM issue above is still worth checking too.
